contracts/fungible-tokens/andromeda-cw20-staking/src/contract.rs

```diff
@@ -238,8 +245,13 @@ fn execute_add_reward_token(
     );
     let mut reward_token = reward_token.check(&env.block, deps.api)?;
     let reward_token_string = reward_token.to_string();
+
     ensure!(
-        !REWARD_TOKENS.has(deps.storage, &reward_token_string),
+        !REWARD_TOKENS.has(deps.storage, &reward_token_string)
+            || !REWARD_TOKENS
+                .load(deps.storage, &reward_token_string)
+                .unwrap()
```

Comment:
It would be better to replace `unwrap()` with `?`

Reply:
Updated